tests/serverless/internal/testsuite/light_runtimes.go

```diff
@@ -92,14 +84,6 @@ func SimpleFunctionTest(restConfig *rest.Config, cfg internal.Config, logf *logr
 				function.UpdateFunction(python312Logger, python312Fn, "Update Python312 Function", runtimes.BasicPythonFunctionWithCustomDependency("Hello From updated python", serverlessv1alpha2.Python312)),
 				assertion.NewHTTPCheck(python312Logger, "Python312 post update simple check through service", python312Fn.FunctionURL, poll, "Hello From updated python"),
 			),
-			executor.NewSerialTestRunner(nodejs18Logger, "NodeJS18 test",
-				configmap.CreateConfigMap(nodejs18Logger, cm, "Create Test ConfigMap", cmData),
-				secret.CreateSecret(nodejs18Logger, sec, "Create Test Secret", secretData),
```

Comment:
We lost this part of the tests. It should be moved to the corresponding NodeJS20 test.